src/components/inputs/layouts/RowLayout.tsx

```diff
@@ -41,13 +41,13 @@ export const RowLayout: FunctionComponent<RowLayoutProps> = (props: RowLayoutPro
     return (
         <Container>
             {
-                props.rows.map((row) => {
+                props.rows.map((row, index) => {
                     const itemWidth = ((100 / row.components.length) + '%');
-                    return <RowContainer>
+                    return <RowContainer key={index}>
                         {
-                            row.components.map((pair) => {
+                            row.components.map((pair, rowIndex) => {
                                 return (
-                                    <ItemContainer width={itemWidth}>
+                                    <ItemContainer key={rowIndex} width={itemWidth}>
```

Comment:
Is it possible to break deeply nested components like this into smaller functions? These functions don't need to be exported, just set in a way to make things easier to read, for instance:
```
function RowContainer(index) {
    return <RowContainer key={index}> {
        row.components.map((pair, rowIndex) => {
            return ItemContainer(pair, rowIndex, itemWidth)
        })
    }
    </RowContainer>
}

function ItemContainer(pair, rowIndex, itemWIdth) {
    return <ItemContainer key={rowIndex} width={itemWidth}>
        <pair.component inputData={pair.inputData}/>
    </ItemContainer>
}
```
Mind that is just matter of a naive opinion, so feel free to disagree or ignore if that doesn't make sense.

Reply:
I think that is a good idea!

Reply:
done